From fe7c16e4fec5c8f17cda0ed1e6da23e7e3317dd5 Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Thu, 24 May 2012 10:42:51 +0800
Subject: [PATCH 1539/2463] ENGR00210559-2 Integrate gpu openGL2.1 and DRM

Add vivante drm modules code and select it in default config. Integrate
these code from vivante.

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 6ec55fd..7e41646 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1693,7 +1693,8 @@ CONFIG_USB_GSPCA=m
 #
 # Graphics support
 #
-# CONFIG_DRM is not set
+CONFIG_DRM=m
+CONFIG_DRM_VIVANTE=m
 # CONFIG_VGASTATE is not set
 # CONFIG_VIDEO_OUTPUT_CONTROL is not set
 CONFIG_FB=y
-- 
1.7.7.4

